content/docs/started/getting-started-multipass.md

```diff
@@ -107,7 +107,8 @@ The current approach leverages ksonnet to setup and install Kubeflow. The kubefl
 $ git clone https://github.com/canonical-labs/kubeflow-tools
 $ kubeflow-tools/install-kubeflow.sh
 ```
-This script will print out the port number for Ambassador and for JupyterHub (Note: you can access JupyterHub through Ambassador).
+This script will print out the port number for Ambassador and for Jupyter notebooks
```

Comment:
Same suggestion as line 10 of this doc.

Reply:
I'm going to stick with "Jupyter notebooks" for simplicity across the docs, as I think that will be clearer to our readers. It may come across as a little confusing to sometimes use "Notebook" and other times "notebook", though I do take your point about correctness here.